src/MCPServer/lib/unit.py

```diff
@@ -77,6 +77,8 @@ def setVariable(self, variable, variableValue, microServiceChainLink):
         upsert `UnitVariable` rows for this unit. This is triggered by
         execution of a workflow chain link of type
         'linkTaskManagerSetUnitVariable'.
+
+        TODO: use Transfer/SIP model methods.
         """
```

Comment:
I didn't want to touch this method before the release. But we could have the method removed as long as the `unit` in MCPServer holds a property with the instance of `Transfer` or `SIP`, so we can call the new `set_active_agent` method.